DesignSpaceEditor.roboFontExt/lib/designSpaceEditorwindow.py

```diff
@@ -951,9 +951,9 @@ def __init__(self, designSpacePath=None):
         ]
         self.w.addToolbar("DesignSpaceToolbar", toolbarItems)
         
-        fileIconWidth = 30
+        fileIconWidth = 50
```

Comment:
I changed these values a bit, because making them resizable made them more responsive to window width. If you open with a small window width, these narrow columns’ headers get lost and replaced with `...`, so I made them a bit wider.

There looks to be a `setMinWidth` method for these table columns, but, like many of the other methods, they weren't working for me. Maybe the vanilla wrapper is blocking some functionality on that level ¯\_(ツ)_/¯